pkg/k8s/errors.go

```diff
@@ -233,3 +233,18 @@ func (e *ImmutableFieldsMutationError) Error() string {
 func NewImmutableFieldsMutationError(immutableFields []string) *ImmutableFieldsMutationError {
 	return &ImmutableFieldsMutationError{immutableFields}
 }
+
+type UpdateNotSupportedError struct {
+	resourceGVK schema.GroupVersionKind
+	namespace   types.NamespacedName
```

Comment:
`namespacedName`?
This is the NamespacedName in the format of `<namespace>/<resourceName>`. It's not just the namespace. In your test log, it is `hcmu7thn7lt2pmq/computetargettcpproxy-hcmu7thn7lt2pmq`.

Reply:
Done